miro_adapter/push_json_to_dynamodb.py

```diff
@@ -40,7 +40,7 @@ def push_to_dynamodb(table_name, collection_name, image_data):
                 Item={
                     'MiroID': image['image_no_calc'],
                     'MiroCollection': collection_name,
-                    'ReindexShard': 'default',
+                    'ReindexShard': collection_name,
```

Comment:
This is the one-line change required to put records in a ReindexShard by collection name.